web/src/components/PreflightResultErrors.jsx

```diff
@@ -151,6 +151,7 @@ class PreflightResultErrors extends Component {
             toggleErrorModal={this.toggleErrorModal}
             err={errorTitle}
             errMsg={errorMsg}
+            appSlug={this.props.appSlug}
```

Comment:
Everywhere else this is using `this.props.match.params.slug`. Is this component being passed that value or something or should this also use params?

Reply:
this component is subcomponent of PreflightResultPage and there I'm passing this.props.match.params.slug
https://github.com/replicatedhq/kots/pull/1616/files#diff-55df8e3fa9c9b98697e9c8f63f1e24f2320ac043ad728aa267e2d6a2bb742842R186

Reply:
great, thanks for clarifying!